packages/shared/src/components/modals/common/Modal.tsx

```diff
@@ -26,6 +26,11 @@ export interface ModalProps extends ReactModal.Props {
   onViewChange?: (view: string) => void;
 }
 
+export type LazyModalCommonProps = Pick<
```

Comment:
@rebelchris I didnt like putting this here, but everywhere else gave me a `dependency cycle` error, so I think this is good enough

Reply:
Should be ok for now 👍